src/realm/table.cpp

```diff
@@ -2395,8 +2420,28 @@ Obj Table::create_object(ObjKey key, const FieldValues& values)
     return obj;
 }
 
+Obj Table::create_embedded_object()
+{
+    if (!m_is_embedded)
+        throw LogicError(LogicError::wrong_kind_of_table);
+    ObjKey key = get_next_key();
+    /*
+    if (auto repl = get_repl())
+        repl->create_object(this, object_id); // FIXME: How is this to work when the object is unlinked atm?
```

Comment:
Perhaps the path should be provided as parameter. We must await the design for the sync protocol. Should that be part of the design doc?